alibi_detect/saving/tensorflow/_loading.py

```diff
@@ -34,7 +34,7 @@
 
 
 def load_model(filepath: Union[str, os.PathLike],
-               load_dir: str = 'model',
+               filename: str = 'model',
```

Comment:
`load_dir` is removed, and any subdirectory (i.e. `model/`) is just added to `filepath` before passing to `load_model`. 

`filename` is added instead, since the legacy saving used to save model's with various filenames such as `encoder.h5` or `model.h5`.